apps/o2-blocks/package.json

```diff
@@ -19,7 +19,7 @@
 	"scripts": {
 		"clean": "npx rimraf dist",
 		"prebuild": "npm run clean",
-		"build": "webpack --config='../../packages/calypso-build/webpack.config.js' --output-path='./dist' --env.WP='true' editor='./src/editor.js' "
+		"build": "cb --output-path='./dist' editor='./src/editor.js' --env.WP"
```

Comment:
If we remove `='true'` from `--env.WP`, it needs to go after `editor='./src/editor.js'`, or Webpack will be confused.

Reply:
Noted that this might be good material for docs https://github.com/Automattic/wp-calypso/issues/31679#issuecomment-484794044